CHANGELOG.md

```diff
@@ -0,0 +1,59 @@
+# EDGEGRID GOLANG RELEASE NOTES
+
+## 2.0.0 (Oct 15, 2020)
```

Comment:
Why do the 1.0.0 Release notes contain 2.0.0 info/reference?

Reply:
Both branches were created at the same time and have the same CHANGELOG file.  I was trying to make v2 changes visible in v1 and this inclusion was a targetted step to help raise awareness of us moving to multiple branches.

Reply:
For context on go branching strategy and support you can read this paper which is really informative and explains how and why go branches are different from other languages.  It is actually quite informative and really helps when working on a large project with multiple "conflicting" indirect dependencies.